wrench/reftests/text/blank.yaml

```diff
@@ -0,0 +1,5 @@
+---
+root:
+  items:
+        - type: rect
+          bounds: [0, 0, 95, 88]
```

Comment:
Oh yes, that does happen. `reftest/gradients/blank.yaml` gets around that by not specifying a `items:`
```yaml
---
root:
```